triveni-codev/GBWebApp-dev
# Justin outputs section bug in removing the associated transport non atsc type 1203813637692182 v2

ASANA ticket: https://app.asana.com/0/1203786645839927/1203813637692182/f

**Description:** Fixed bug in removing the associated transport NON-ATSC type
- Amended the code which will retrieve the corresponding transport in each Output after removing it ~ whether it's in the ADD, EDIT, ADDEDIT, PREVIOUS, or NEXT logic functions

**TO TEST:**
- Create/Update an Output via dirty change or server using only the ff. Output Types (ATSC3, Carousel, Thomson, The 3 Ericssons)
- Conduct a play-by-play scenario in associating a Transport in an Output by the ff.:
      *adding and removing a transport (vice versa)
      *updating and removing a transport (vice versa)
      *clicking next and previous (vice versa)
      *save and cancel
      *ADD, EDIT, ADDEDIT play-by-play scenarios
- Counter-check via JavaApp
- Test BIG COMMIT

NOTE: Basically, this PR revolves only in the Transport Section inside an Output. Test the entire play-by-play scenarios of associating a Transport in an Output ~ excluding this scenario: https://app.asana.com/0/1203786645839927/1203834652047700/f

Also, OOS bugs can be logged here https://app.asana.com/0/1203786645839927/1203834652047708/f

Exclusively requesting for a review on this Darl @darlp5589 , as you already have an initial knowledge on how to test this after reviewing my other PRs earlier. Thank you!

cc: @triveni-codev
